src/styles/components/buttons/styles.less

```diff
@@ -102,149 +107,132 @@ components/buttons.less
         &:last-child {
           margin-bottom: 0;
         }
-      }
-    }
-  }
 
-  /* -----------------------------------------------------------------------------
-  --------------------------------------------------------------------------------
+      }
 
-  Button Group
+    }
 
-  --------------------------------------------------------------------------------
-  ----------------------------------------------------------------------------- */
+  }
 
   .button-group {
-
     margin-bottom: @base-spacing-unit * 0.5;
 
     .dot {
```

Comment:
The `.dot` and `.badge` styles can be removed now, as the new CNVS applies consistent spacing between child elements of a button.  NOTE: the text in the button needs to be wrapped in `<span>` tag for this to work.